cesql/spec.md

```diff
@@ -278,6 +305,8 @@ For example, the pattern `_b*` will accept values `ab`, `abc`, `abcd1` but won't
 Both `%` and `_` can be escaped with `\`, in order to be matched literally. For example, the pattern `abc\%` will match
 `abc%` but won't match `abcd`.
 
+The pattern matching MUST be case sensitive.
```

Comment:
As above, it might be nice to say it's "case sensitive" in each row rather than splitting the full definition between the table and a sentence 5 paragraphs away.